packages/apollo-graphql/src/__tests__/signature.test.ts

```diff
@@ -13,7 +13,7 @@ import {
 // breaks if you turn it off in tests.
 disableFragmentWarnings();
 
-describe('aggressive signature', () => {
+describe.only('aggressive signature', () => {
```

Comment:
This is unrelated to the actual work on the PR, but I suggest [`eslint-plugin-jest`'s `no-focused-tests`](https://github.com/jest-community/eslint-plugin-jest/blob/HEAD/docs/rules/no-focused-tests.md) to make CI fail when you have focused tests (meaning `.only`)

Reply:
Resolved in 077e2c7621ea02a52ea3aa2a4f5416e5287829a0